### ISSUE TITLE
When clustered owncloud setup install task runs only once for the first node

### ISSUE THREAD

**Issue by JKawohl** [owncloud-ansible/owncloud#9]

**Comment by xoxys** [comment]
This happens for tasks that are using the `occ` tool. If you don't have configured a shared storage the config file exists only on the first oC host. While the `occ` is looking for the `installed` flag in the config.php file these tasks will fail.

For a clustered setup some kind of shared storage is a hard requirement. We need to document it.

**Comment by enbrnz** [comment]
Closed as it is already set as `wontfix`, added as TODO to our documentation issue: https://github.com/owncloud-ansible/owncloud-ansible.github.io/issues/3#issue-732150300
